src/lib/operators/get_table.cpp

```diff
@@ -160,6 +160,7 @@ std::shared_ptr<const Table> GetTable::_on_execute() {
     } else {
       auto output_segments = Segments{stored_table->column_count() - _pruned_column_ids.size()};
       auto output_segments_iter = output_segments.begin();
+      auto output_indices = Indices{};
```

Comment:
Above you use the plural `indexes`. Here it is `indices`. There might be further inconsistencies in the code base, but let's try to make it more consistent. When I heard native speakers talking about that topic, they always used `indexes`.

Reply:
Make the linter complain about indices?

Reply:
I believe for the plural of a loop/array/vector index, indices is fine. Maybe we should consult a native speaker.